### PR TITLE
feat: added continents function

### PR DESCRIPTION
**Pull request by charan-003**
added module for continents and data

### CODE REVIEW

File: src/modules/location.cpp [hunk 1]
```diff
@@ -21,73 +23,104 @@ namespace faker::location
...
+        case faker::Locale::ca_ES:
+        case faker::Locale::es_ES:
+            return spainAddresses;
+        case faker::Locale::pt_BR:
+            return brazilAddresses;
+        case faker::Locale::fi_FI:
+            return finlandAddresses;
+        case faker::Locale::et_EE:
+            return estoniaAddresses;
+        case faker::Locale::en_GB:
```
Comment (cieslarmichal) [inline src/modules/location.cpp:75]: Why did you add faker prefix? Its all in faker namespace so no need

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

File: src/modules/location_data.h [hunk 2]
```diff
@@ -4,6 +4,12 @@
 #include <span>
 #include <string_view>
 
+#include <unordered_map>
+namespace faker::location
+{
```
Comment (cieslarmichal) [inline src/modules/location_data.h:9]: It should be declared in location.h in include directory

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

**Review by charan-003 — COMMENTED** [review]
.

**Comment by charan-003** [comment]
i see 2 test cases are failing, where can i check that

**Comment by cieslarmichal** [comment]
Well its best to run it locally

**Comment by cieslarmichal** [comment]
If you need any help with running tests let me know, best to reach faker developers is by our discord channel, link in readme

**Comment by charan-003** [comment]
@cieslarmichal so there are [  FAILED  ] LocationTest.shouldGenerateCorrectContinentForKnownCountry
[  FAILED  ] LocationTest.shouldReturnUnknownForUnmappedCountry
[  FAILED  ] LocationTest.shouldGenerateRandomContinent

these test cases failing and I'm not sure what's wrong with it

**Pull request closed without merging**